403.php

```diff
@@ -0,0 +1,27 @@
+<?php
+    include '_templates/sitewide.php';
+    $page['name'] = '403';
+    $page['title'] = 'Forbidden &sdot; elementary';
```

Comment:
I think the page title should probably be "Access Denied", to match how the others are formatted (in a short but readable way).